.github/workflows/push_openapi_spec_to_readme.yml

```
@@ -0,0 +1,36 @@
+name: Build and sync docs on PR merge or pushes to master branch
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+
+jobs:
+  rdme-openapi-pr:
+    # Only trigger this job if it was a pushed event OR a PR merged event
+    if: github.event_name == 'push' || github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Authenticate to AWS
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::403483446840:role/autogen_github_actions_beta_push_openapi_spec_to_readme
+      - name: Load secrets
+        uses: aws-actions/aws-secretsmanager-get-secrets@v2
+        with:
+          secret-ids: README_API,prod/github_push_openapi_spec_to_readme/readme_api_key
```

since this is a role in `beta`, you should say `beta` here 😄 you also want the role name to match exactly 
```suggestion
          secret-ids: README_API,beta/github_actions_push_openapi_spec_to_readme/readme_api_key
```

---

Ah ok, this was why I asked about this and Matt said **_"That looks right to me"_**.  But I think Matt didn't know that we added this to `beta`.